### PR TITLE
[Bug] Remove license concepts in packages/osd-opensearch

### PR DESCRIPTION
**Pull request by ananzh**
### Description

Currenlty, /packages/osd-opensearch/README.md still have license concepts. This PR cleans the concepts.

### Partially Resolved:
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/592

Signed-off-by: Anan Zhuang <ananzh@amazon.com>

 
### Check List
- [ ] New functionality includes testing.
  - [x] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by kavilla — DISMISSED** [review]

File: packages/osd-opensearch/README.md [hunk 1]
```diff
@@ -9,9 +9,9 @@ To run, go to the OpenSearch Dashboards root and run `node scripts/opensearch --
 
 ### Examples
 
-Run a snapshot install with a trial license
+Run a snapshot install with an oss license
 ```
-node scripts/opensearch snapshot --license=trial
+node scripts/opensearch snapshot --license=oss
```
Comment (mihirsoni) [inline packages/osd-opensearch/README.md:14]: Are these even valid we should remove it.

Comment (ananzh) [reply packages/osd-opensearch/README.md:14]: will do

**Review by kavilla — APPROVED** [review]

**Review by mihirsoni — APPROVED** [review]
LGTM !! 
Thanks for changes, I think we should have issue to piggy back on this later. I am unsure if this working at all.

**Comment by ananzh** [comment]
> LGTM !!
> Thanks for changes, I think we should have issue to piggy back on this later. I am unsure if this working at all.

Thanks @mihirsoni I will create an issue to double check whether snapshot cmd working

**Comment by ananzh** [comment]
Follow up issue: #619

**Pull request merged**